`cstdint` not included in `VectorArrayAPI.hpp`

Making on Arch gives the following error.

```
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I../../src -I../../src/4ti2 -I/home/kss/M2/M2/include -I/home/kss/M2/M2/include -I/home/kss/M2/M2/usr-host/include -isystem /usr/include/libxml2 -DSING_NDEBUG -DOM_NDEBUG -DNDEBUG -I/usr/include/cddlib -I/usr/include/eigen3 -I/usr/include/python3.11 -I/usr/include -DBOOST_STACKTRACE_LINK -I./.. -D__STDC_LIMIT_MACROS -O3 -fomit-frame-pointer -malign-double -fstrict-aliasing -ffast-math -Wall -std=c++0x -MT libzsolve_la-RelAPI.lo -MD -MP -MF .deps/libzsolve_la-RelAPI.Tpo -c RelAPI.cpp  -fPIC -DPIC -o .libs/libzsolve_la-RelAPI.o
In file included from ../../src/zsolve/RelAPI.h:26,
                 from RelAPI.cpp:23:
../../src/zsolve/VectorArrayAPI.hpp: In function 'void _4ti2_zsolve_::convert(const T1&, T2&) [with T1 = long int; T2 = int]':
../../src/zsolve/VectorArrayAPI.hpp:79:14: error: 'INT32_MIN' was not declared in this scope; did you mean 'INT_MIN'?
   79 |     if (v1 < INT32_MIN || v2 > INT32_MAX) {
      |              ^~~~~~~~~
      |              INT_MIN
../../src/zsolve/VectorArrayAPI.hpp:79:32: error: 'INT32_MAX' was not declared in this scope
   79 |     if (v1 < INT32_MIN || v2 > INT32_MAX) {
      |                                ^~~~~~~~~
../../src/zsolve/VectorArrayAPI.hpp:31:1: note: 'INT32_MAX' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   30 | #include <cstdlib>
  +++ |+#include <cstdint>
   31 |
make[6]: *** [Makefile:879: libzsolve_la-RelAPI.lo] Error 1
make[6]: Leaving directory '/home/kss/M2/M2/libraries/4ti2/build/4ti2-1.6.9/src/zsolve'
```

I included the suggested library, and it worked. Does it have something to do with the the C++ standards, 11 vs 17 ?

**edit**

I should've included my gcc version, 13.1.1

--- a/M2/cmake/build-libraries.cmake
+++ b/M2/cmake/build-libraries.cmake
@@ -823,12 +823,13 @@ set(4ti2_PROGRAMS
   qsolve rays walk zbasis zsolve hilbert graver ppi genmodel gensymm output)
 list(TRANSFORM 4ti2_PROGRAMS PREPEND ${M2_HOST_PREFIX}/bin/ OUTPUT_VARIABLE 4ti2_PROGRAMS)
 ExternalProject_Add(build-4ti2
-  URL               https://github.com/4ti2/4ti2/releases/download/Release_1_6_9/4ti2-1.6.9.tar.gz
-  URL_HASH          SHA256=3053e7467b5585ad852f6a56e78e28352653943e7249ad5e5174d4744d174966
+  URL               https://github.com/4ti2/4ti2/releases/download/Release_1_6_10/4ti2-1.6.10.tar.gz
+  URL_HASH          SHA256=f7c191beb14246b643e4fd5b18b53d9966693b9e6d3a569441a0e3ca14b1a86b
   PREFIX            libraries/4ti2
   SOURCE_DIR        libraries/4ti2/build
   DOWNLOAD_DIR      ${CMAKE_SOURCE_DIR}/BUILD/tarfiles
   BUILD_IN_SOURCE   ON
+  PATCH_COMMAND     mkdir swig # needed because the tar doesn't have a swig directory but 4ti2's Makefile.am seems to need it
   CONFIGURE_COMMAND autoreconf -vif
             COMMAND ${CONFIGURE} --prefix=${M2_HOST_PREFIX}
                       #-C --cache-file=${CONFIGURE_CACHE}

--- a/M2/libraries/4ti2/Makefile.in
+++ b/M2/libraries/4ti2/Makefile.in
@@ -1,13 +1,12 @@
 # temporary beta testing version
-VERSION = 1.6.9
+VERSION = 1.6.10
 TARDIR = 4ti2-$(VERSION)
 
 # PATCHFILE = @abs_srcdir@/patch-$(VERSION)
-PRECONFIGURE = autoreconf -vif
+PRECONFIGURE = mkdir -p swig && autoreconf -vif
 
-# URL = http://www.4ti2.de/version_$(VERSION)
-# URL = https://github.com/4ti2/4ti2/releases/download/Release_1_6_9/4ti2-1.6.9.tar.gz
-URL = http://macaulay2.com/Downloads/OtherSourceCode
+VERSION_ = $(shell echo $(VERSION) | sed 's/\./_/g')
+URL = https://github.com/4ti2/4ti2/releases/download/Release_$(VERSION_)
 
 ifeq (@ENABLE_STRIP@,yes)
 INSTALLTARGET = install-strip

